camera_driver:
  ros__parameters:
    camera_id: 19443010C1354D1300 # A string of 18 characters long, mxid of depthai oak camera
    camera_frame: camera_link
    pointcloud_topic: /camera/points/raw
    image_topic: /camera/image/raw
    frequency: 30
